inc/search.class.php

```diff
@@ -6124,6 +6126,7 @@ static function giveItem($itemtype, $ID, array $data, $meta = 0,
                                                                    'display' => false]));
                   return $out;
                }
+               break;
```

Comment:
Not sure

Reply:
`break` is OK here.

There was previously a `return` instruction that has been removed, so code was never made to continue in next case.

See https://github.com/glpi-project/glpi/commit/05be96466f63dfd68ab100f37a3b8791389053de